lib/Screens/Login.dart

```diff
@@ -1,158 +1,159 @@
+import 'package:food_ordering/utils/utils.dart';
 
-class SignIn extends StatelessWidget {
+class Login extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-        image: DecorationImage(
-          // Background Image
-            image: AssetImage('Assets/pizza.jpg'), 
-            fit: BoxFit.cover),
-      ),
-      child: Scaffold(
-        backgroundColor: Colors.transparent,
-              body: Container(
-          width: MediaQuery.of(context).size.width,
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(20),
-            color: Colors.transparent,
-          ),
-          child: Padding(
-            padding: const EdgeInsets.all(23.0),
-            child: ListView(
-              shrinkWrap: true,
-              children: <Widget>[
-                SizedBox(height: 200,),
-                Form(
-                  child: Column(
-                    children: <Widget>[
-                      Padding(
-                        padding: const EdgeInsets.fromLTRB(0, 20, 0, 20),
-                        child: TextFormField(
+    return Scaffold(
+      body: Container(
+        decoration: BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage('assets/pizza.jpg'), fit: BoxFit.cover),
+        ),
+        child: Scaffold(
+          backgroundColor: Colors.transparent,
+          body: Container(
+            width: MediaQuery.of(context).size.width,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(20),
+              color: Colors.transparent,
+            ),
+            child: Padding(
+              padding: const EdgeInsets.all(23.0),
+              child: ListView(
+                shrinkWrap: true,
+                children: <Widget>[
+                  SizedBox(
+                    height: 200,
+                  ),
+                  Form(
+                    child: Column(
+                      children: <Widget>[
+                        Padding(
+                          padding: const EdgeInsets.symmetric(vertical: 20),
+                          child: TextFormField(
+                            style: TextStyle(
+                              color: Colors.white,
+                            ),
+                            decoration: InputDecoration(
+                              enabledBorder: const UnderlineInputBorder(
+                                borderSide: const BorderSide(
+                                  color: Colors.white,
+                                ),
+                              ),
+                              labelText: Strings.username,
+                              labelStyle:
+                                  TextStyle(fontSize: 15, color: Colors.white),
+                            ),
+                          ),
+                        ),
```

Comment:
Consider extracting this decoration or even whole TextFormField in separate file so it can be reused in many places. What do you think about this?

Reply:
Textstyle is not same everywhere

Reply:
I mean InputDecoration(...) It's same for username and password so it's repeated in 2 places.

Reply:
So for that I think I should make a function below instead of declaring inside another file

Reply:
you can declare it inside this file for now if you prefer. But why in function instead some final/const variable?

Reply:
Beacause we have two types of  textFields, one with the label of username and second on with the label if password. So if I declare a function, label can be sent as a parameter.

Reply:
In flutter everything is a widget, so that means we have to compose widgets inorder to build something. Make a stateless widget and pass parameters in constructor.

Reply:
We can also do the same thing by making a function and passing the rquired parameters,
No need of a stless widget in this case acc. to me

Reply:
What if we need the same widget  in another class?

Reply:
Ya then you are right, but we may not require the same widget somewhere else as of now

Reply:
@Maadhav you are right, my mistake, inputDecorations are different, it can be left as it is in this case